upgrade/UPGRADE-v9.0.0-dev.md

```diff
@@ -62,6 +62,12 @@ There you can find links to upgrade notes for other versions too.
     - copy necessary type definitions
         [Query.types.yml from Github](https://github.com/shopsys/shopsys/blob/9.0/project-base/src/Shopsys/ShopBundle/Resources/graphql-types/Query.types.yml) to `src/Shopsys/ShopBundle/Resources/graphql-types/Query.types.yml`
         [Category.types.yml from Github](https://github.com/shopsys/shopsys/blob/9.0/project-base/src/Shopsys/ShopBundle/Resources/graphql-types/Category.types.yml) to `src/Shopsys/ShopBundle/Resources/graphql-types/Category.types.yml`
+        [Availability.types.yml from Github](https://github.com/shopsys/shopsys/blob/9.0/project-base/src/Shopsys/ShopBundle/Resources/graphql-types/Availability.types.yml) to `src/Shopsys/ShopBundle/Resources/graphql-types/Availability.types.yml`
+        [Flag.types.yml from Github](https://github.com/shopsys/shopsys/blob/9.0/project-base/src/Shopsys/ShopBundle/Resources/graphql-types/Flag.types.yml) to `src/Shopsys/ShopBundle/Resources/graphql-types/Flag.types.yml`
+        [Money.types.yml from Github](https://github.com/shopsys/shopsys/blob/9.0/project-base/src/Shopsys/ShopBundle/Resources/graphql-types/Money.types.yml) to `src/Shopsys/ShopBundle/Resources/graphql-types/Money.types.yml`
+        [Price.types.yml from Github](https://github.com/shopsys/shopsys/blob/9.0/project-base/src/Shopsys/ShopBundle/Resources/graphql-types/Price.types.yml) to `src/Shopsys/ShopBundle/Resources/graphql-types/Price.types.yml`
+        [Product.types.yml from Github](https://github.com/shopsys/shopsys/blob/9.0/project-base/src/Shopsys/ShopBundle/Resources/graphql-types/Product.types.yml) to `src/Shopsys/ShopBundle/Resources/graphql-types/Product.types.yml`
+        [Unit.types.yml from Github](https://github.com/shopsys/shopsys/blob/9.0/project-base/src/Shopsys/ShopBundle/Resources/graphql-types/Unit.types.yml) to `src/Shopsys/ShopBundle/Resources/graphql-types/Unit.types.yml`
```

Comment:
You have added or changed some tests, this needs to be described here too and you have also changed shopsys_frontend_api.yml file

Reply:
Ok, but I don't understand why? On line 72 is text `copy necessary configuration [shopsys_frontend_api.yml from Github](https://github.com/shopsys/shopsys/blob/9.0/project-base/app/config/packages/shopsys_frontend_api.yml) to app/config/packages/shopsys_frontend_api.yml` and on line 73 is text `    - copy [tests for FrontendApiBundle from Github](https://github.com/shopsys/shopsys/tree/9.0/project-base/tests/FrontendApiBundle) to your tests folder` and this two steps includes my changes too. Or not?

Reply:
Ok